aws-cpp-sdk-core/source/http/windows/WinHttpSyncHttpClient.cpp

```diff
@@ -133,6 +125,15 @@ void* WinHttpSyncHttpClient::OpenRequest(const Aws::Http::HttpRequest& request,
     HINTERNET hHttpRequest = WinHttpOpenRequest(connection, StringUtils::ToWString(HttpMethodMapper::GetNameForHttpMethod(request.GetMethod())).c_str(),
         wss.c_str(), nullptr, nullptr, accept, requestFlags);
 
+    if (!m_verifySSL)
```

Comment:
After the tcp connection has already been made? We're at layer 7 already before setting these options.